substation-diagram/substation-diagram-core/src/test/resources/substDiag.svg

```diff
@@ -75,20 +75,21 @@
             <line y2="615.0" x1="2150.0" class="grid" x2="2150.0" transform="translate(20.0,50.0)" y1="30.0"/>
             <line y2="615.0" x1="2200.0" class="grid" x2="2200.0" transform="translate(20.0,50.0)" y1="30.0"/>
             <line y2="615.0" x1="2250.0" class="grid" x2="2250.0" transform="translate(20.0,50.0)" y1="30.0"/>
+            <line y2="615.0" x1="2300.0" class="grid" x2="2300.0" transform="translate(20.0,50.0)" y1="30.0"/>
         </g>
         <g class="substation-diagram BUSBAR_SECTION bbs1" id="bbs1" transform="translate(80.0,360.0)">
-            <line y2="0" x1="0" class="bus_vl1" x2="430.0" y1="0"/>
+            <line y2="0" x1="0" class="bus_vl1" x2="480.0" y1="0"/>
             <text x="-5.0" font-size="8" y="-5.0" class="component-label" font-family="Verdana" transform="rotate(0,0,0)">bbs1</text>
         </g>
-        <g class="substation-diagram BUSBAR_SECTION bbs2" id="bbs2" transform="translate(580.0,360.0)">
+        <g class="substation-diagram BUSBAR_SECTION bbs2" id="bbs2" transform="translate(630.0,360.0)">
             <line y2="0" x1="0" class="bus_vl1" x2="280.0" y1="0"/>
             <text x="-5.0" font-size="8" y="-5.0" class="component-label" font-family="Verdana" transform="rotate(0,0,0)">bbs2</text>
         </g>
         <g class="substation-diagram BUSBAR_SECTION bbs3" id="bbs3" transform="translate(80.0,385.0)">
-            <line y2="0" x1="0" class="bus_vl1" x2="430.0" y1="0"/>
+            <line y2="0" x1="0" class="bus_vl1" x2="480.0" y1="0"/>
```

Comment:
How do you explain this change in the rendering?

Reply:
In order to improve the lines and conditions coverage, I enriched some tests (TestCase11SubstationGraph, TestCase12GraphWith3WT), which explains the difference in the generated SVG test files.